pyartcd/pyartcd/pipelines/build_microshift_bootc.py

```
@@ -427,6 +427,27 @@ async def _get_microshift_rpm_commit(self) -> str:
             f"The NVR release fields do not contain a recognizable commit hash."
         )
 
+    def _get_assembly_label_value(self) -> Optional[str]:
+        """Compute the value for the assembly Dockerfile label.
+
+        - STANDARD (e.g. assembly ``4.18.1``): ``v4.18.1``
+        - CANDIDATE / PREVIEW (e.g. assembly ``rc.1``): ``v4.22.0-rc.1``
+        - CUSTOM: raises – not supported for microshift-bootc
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)